pkg/rpc/auth_tenant.go

```diff
@@ -79,7 +80,11 @@ func (a tenantAuth) tenantFromCert(ctx context.Context) (roachpb.TenantID, error
 	}
 
 	commonName := tlsInfo.State.PeerCertificates[0].Subject.CommonName
-	tenID, err := strconv.ParseUint(commonName, 10, 64)
+	const prefix = "tenant_"
```

Comment:
I will change this so we don't rely on prefix (that would make hard to distinguish tenant from an SQL user named tenant_33). The type of the CN will be in OU. I.e. `OU=Tenants CN=33`